docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

```diff
@@ -259,59 +316,72 @@ Converts newline characters to `<br>` tags.
 
 [See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/nl2br.js).
 
-[See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=nl2br).
-
 ### {{pre}}

```

Comment:
@junedkazi Can you please provide an example for this helper?

Reply:
https://github.com/bigcommerce/paper-handlebars/blob/ce4c27f567fb7835c1205ab54bd29913bcdc4950/spec/helpers/pre.js#L20

We should also call out that this will escape text.